modules/system/classes/UpdateManager.php

```diff
@@ -365,21 +365,29 @@ public function uninstall()
     }
 
     /**
-     * Asks the gateway for the lastest build number and stores it.
+     * Asks the composer/gateway for the lastest build number and stores it.
```

Comment:
@Samuell1 Please change the comment to the following:
```suggestion
     * Asks the gateway (or Composer) for the latest build number and stores it in the database.
```